src/App.scss

```diff
@@ -0,0 +1,9 @@
+* {
+  text-align: center;
+  padding: 5px;
```

Comment:
not sure all elements need to have such rules :)

Reply:
agree :)